src/test/kotlin/cacophony/automata/DFAEquivalenceTest.kt

```diff
@@ -0,0 +1,398 @@
+            )
+        assertTrue(areEquivalent(large, small))
+    }
+
+    @Test
+    fun `test two DFAs for the same language are equivalent when one has multiple accepting states`() {
+        val singleAccepting =
+            createDFA(
+                1,
+                mapOf(
+                    Pair(1, 'a') to 2,
+                    Pair(1, 'b') to 3,
+                    Pair(1, 'c') to 4,
+                    Pair(1, 'd') to 5,
+                    Pair(2, 'a') to 6,
+                    Pair(3, 'a') to 6,
+                    Pair(4, 'a') to 6,
+                    Pair(5, 'a') to 6,
+                ),
+                setOf(6),
+            )
+        val multipleAccepting =
+            createDFA(
+                1,
+                mapOf(
+                    Pair(1, 'a') to 2,
+                    Pair(2, 'a') to 3,
+                    Pair(1, 'b') to 4,
+                    Pair(4, 'a') to 5,
+                    Pair(1, 'c') to 6,
+                    Pair(6, 'a') to 7,
+                    Pair(1, 'd') to 8,
+                    Pair(8, 'a') to 9,
+                ),
+                setOf(3, 5, 7, 9),
+            )
+        assertTrue(areEquivalent(singleAccepting, multipleAccepting))
+    }
+
+    @Test
+    fun `test two DFAs for the same language are equivalent when both have multiple accepting states`() {
+        val first =
+            createDFA(
+                1,
+                mapOf(
+                    Pair(1, 'a') to 3,
+                    Pair(1, 'b') to 2,
+                    Pair(2, 'a') to 3,
+                    Pair(3, 'a') to 4,
+                    Pair(3, 'b') to 5,
+                ),
+                setOf(3, 4, 5),
+            )
+        val second =
+            createDFA(
+                1,
+                mapOf(
+                    Pair(1, 'a') to 2,
+                    Pair(1, 'b') to 4,
+                    Pair(2, 'a') to 3,
+                    Pair(2, 'b') to 8,
+                    Pair(4, 'a') to 5,
+                    Pair(5, 'a') to 7,
+                    Pair(5, 'b') to 6,
+                ),
+                setOf(2, 3, 5, 6, 7, 8),
+            )
+        assertTrue(areEquivalent(first, second))
+    }
+
+    @Test
+    fun `test DFA with no accepting states is equivalent to one with unreachable accepting states`() {
+        val noAcceptingStates =
+            createDFA(
+                1,
+                mapOf(
+                    Pair(1, 'a') to 1,
+                    Pair(1, 'b') to 1,
+                ),
+                setOf(),
+            )
+        val unreachableAcceptingStates =
+            createDFA(
+                1,
+                mapOf(
+                    Pair(1, 'a') to 1,
+                    Pair(1, 'b') to 1,
+                    Pair(2, 'a') to 1,
+                    Pair(3, 'b') to 1,
+                    Pair(4, 'a') to 2,
+                    Pair(4, 'b') to 3,
+                ),
+                setOf(),
```

Comment:
Accepting state is not _unreachable_, it just doesn't exist

Reply:
They were supposed to be {2, 3, 4}
Fixed, test passes